src/components/scenes/CreateWalletImportScene.tsx

```diff
@@ -57,6 +66,7 @@ const CreateWalletImportComponent = (props: Props) => {
     const promises = pluginIds.map(
       async pluginId =>
         await currencyConfig[pluginId].importKey(cleanImportText).catch(e => {
+          showError(e)
```

Comment:
Are there specific error codes thrown here that we can translate. If so show the translation.

Reply:
Unfortunately, no, the currency plugins all use generic errors